com.unity.formats.alembic/package.json

```diff
@@ -2,7 +2,8 @@
   "dependencies": {
     "com.unity.timeline": "1.0.0",
     "com.unity.modules.unityanalytics": "1.0.0",
-    "com.unity.modules.cloth": "1.0.0"
+    "com.unity.modules.cloth": "1.0.0",
+    "com.unity.collections" : "1.0.0"
```

Comment:
FixedString128 is the one that forces us to use 23.1 and "com.unity.collections"